pinot-plugins/pinot-minion-tasks/pinot-minion-builtin-tasks/src/main/java/org/apache/pinot/plugin/minion/tasks/segmentgenerationandpush/SegmentGenerationAndPushTaskUtils.java

```diff
@@ -67,8 +66,7 @@ static PinotFS getOutputPinotFS(Map<String, String> taskConfigs, URI fileURI)
       pinotFS.init(fsProps);
       return pinotFS;
```

Comment:
how about add a public method PinotFSFactory.createNewInstance(fileURIScheme, fsClass, fsProps) or make PinotFSFactory. createPinotFSInstance() public and call it here?